crunchydb/high-availablility/README.md

```diff
@@ -12,6 +12,33 @@ In this example, my license plate is `be1c6b`.
 
 `PostgresCluster` has the sample PostgresCluster with 3 replicas, backups, and monitoring.
 
+## Images
+
+The PostgresCluster should include the version specifications for PostgreSQL and if required PostGIS. The reference to the image should always be specified to ensure that it can be found.
+
+```
+spec:
+  postgresVersion: 15
+  postGISVersion: "3.3"
+  image: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres-gis:ubi8-15.2-3.3-0
+  imagePullPolicy: IfNotPresent
+```
+
+The following images are available in the artifacts.developer.gov.bc.ca/bcgov-docker-local image repository.
+
+* Postgres
+    * 13 ?????

```

Comment:
```suggestion
    * 13
       * crunchy-postgres:centos8-13.5-0
```

Reply:
We should probably also include a note that you should definitely NOT use this image - it doesn't work well with the new pgbackrest image, so it should really only be used if you need an interim option for a migration or something, and intend to upgrade to 14 or 15 very quickly.